codeforboston/maple
# Migrating OurPartnersContent to Bootstrap

# Summary

This PR migrates the OurPartnersCardContent component to use Bootstrap - resolving #1452 

# Checklist

- [na] On the frontend, I've made my strings translate-able.
- [na] If I've added shared components, I've added a storybook story.

# Screenshots

Current Page:

<img width="1096" alt="Screenshot 2024-04-19 at 10 56 38 PM" src="https://github.com/codeforboston/maple/assets/2694761/2878bd6b-15f1-47ee-8363-f50da90a8f34">
<img width="424" alt="Screenshot 2024-04-19 at 10 57 17 PM" src="https://github.com/codeforboston/maple/assets/2694761/4792e2a5-d65f-42f7-ab0c-86d5ca1ef5ae">

Updated Page:

<img width="1092" alt="Screenshot 2024-04-19 at 10 56 48 PM" src="https://github.com/codeforboston/maple/assets/2694761/1bade38e-9ef2-45ef-baf8-54237950be8b">
<img width="400" alt="Screenshot 2024-04-19 at 10 57 30 PM" src="https://github.com/codeforboston/maple/assets/2694761/71576671-664e-4737-b935-8fb6662cafb8">


# Known issues

There are some minor style differences between this and the current prod page (this has the correct background, slightly more readable font, standard link decorations on hyperlinks) - I think this looks good and is more consistent with our standard bootstrap styles, but wanted to call it out.

# Steps to test/reproduce

_For each feature or bug fix, create a step by step list for how a reviewer can test it out. E.g.:_

1. Go to the Partners page
2. Verify that the content renders as expected
